src/supermarket/app/views/layouts/application.html.erb

```diff
@@ -18,11 +18,15 @@
       <%= stylesheet_link_tag "//fonts.googleapis.com/css?family=Montserrat:400,700" %>
     <% end %>
     <%= csrf_meta_tags %>
-    <% unless air_gapped? %>
```

Comment:
Is this check no longer required ?

Reply:
I removed it by mistake adding it back. Thanks